addons/sale/views/sale_portal_templates.xml

```diff
@@ -130,18 +130,50 @@
                 <t t-call="portal.portal_record_sidebar" id="sale_order_portal_sidebar">
                     <t t-set="classes" t-value="'col-lg-4 col-xxl-3 d-print-none'"/>
-
-                    <t t-set="title">
-                        <h2 t-field="sale_order.amount_total" data-id="total_amount" class="mb-0 text-break"/>
+                    <t t-if="sale_order.amount_total == sale_order._get_prepayment_required_amount()">
+                        <t t-set="title">
+                            <h2 t-field="sale_order.amount_total" data-id="total_amount" class="mb-0 text-break" />
+                        </t>
+                    </t>
+                    <t t-else="">
+                        <t t-set="title">
+                            <h4>
+                                <div>
+                                    Down payment
+                                </div>
+                                <h3>
+                                    <t t-out="sale_order._get_prepayment_required_amount()"
+                                       data-id="total_amount"
+                                       class="mb-0 text-break"
+                                       t-options="{'widget': 'monetary', 'display_currency': sale_order.currency_id}"
+                                    />
+                                    (<b t-esc="sale_order.prepayment_percent * 100"/>%)
+                                </h3>
+                            </h4>
+                        </t>
                     </t>
                     <t t-set="entries">
+                        <t t-if="sale_order.state in 'draft,sent' and sale_order.require_payment">
+                            <t t-if="sale_order.amount_paid > 0">
+                                <div class="alert alert-success" role="alert">
+                                    Amount paid:
+                                    <span t-field="sale_order.amount_paid"
+                                          t-options="{'widget': 'monetary', 'display_currency': sale_order.currency_id}"
+                                    />
+                                </div>
+                            </t>
+                            <div class="alert alert-warning" role="alert">
+                                Left to pay:
+                                <span t-field="sale_order.amount_due"/>
+                            </div>
+                        </t>
                         <div class="d-flex flex-column gap-4 mt-3">
                             <div class="d-flex flex-column gap-2" id="sale_order_sidebar_button">
-                                <a t-if="sale_order._has_to_be_signed()" role="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#modalaccept" href="#">
+                                <a t-if="sale_order._has_to_be_signed() and not link_amount" role="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#modalaccept" href="#">
```

Comment:
It was discussed with Dala at some point and the conclusion was that when it is with payment link, the sign part is not shown and only the pay part is shown.

Reply:
Got it! Could you mention it in dev notes, the decision, and the reasoning behind it, just to remember it and in case someone else will ask the same question?